index.js

```diff
@@ -11,5 +14,10 @@ module.exports = {
     './possible-errors.js',
     './variables.js',
   ],
-  rules: {}
+  rules: {
+    'prettier/prettier': ['error', {
+      'singleQuote': true,
+      'printWidth': 140
```

Comment:
What led y’all to this specific line length? Like the docs say, I feel like I tend to see 100 or 120 in other coding style guides (or even the default of 80)

Reply:
This is slightly outdated. It'll be 100 which is used in the example life-web pr.

Reply:
yup my bad, let me update!